pastas/rfunc.py

```diff
@@ -52,19 +52,34 @@
 class RfuncBase:
     _name = "RfuncBase"
 
-    def __init__(self, **kwargs) -> None:
-        self.up = True
-        self.meanstress = 1
-        self.cutoff = 0.999
+    def __init__(
+        self, up: bool = True, meanstress: float = 1.0, cutoff: float = 0.999, **kwargs
+    ) -> None:
+        self.up = up
+        self.meanstress = meanstress
+        self.cutoff = cutoff
         self.kwargs = kwargs
 
-    def _set_init_parameter_settings(
+    def _update_rfunc_settings(
```

Comment:
The response function notebook contains a reference to this function still.

Reply:
Yes, this is for the next step when working on the documentation, merging these two notebooks eventually.